examples/architext.py

```diff
@@ -9,22 +9,33 @@ def reward_fn(samples):
     return [-sample.count(":") for sample in samples]
```

Comment:
Do we have a wandb example run for this?

Reply:
I was waiting for hyperparam's search to make one!